iree/turbine/kernel/wave/utils.py

```diff
@@ -818,6 +818,7 @@ def graph_copy(graph: fx.Graph) -> tuple[fx.Graph, dict[fx.Node, fx.Node]]:
     for node in graph.nodes:
         custom = get_custom(node)
         new_node = custom.copy(
+            new_name=node.name,
```

Comment:
This is a utility function, so it could potentially be used outside of scheduling (although currently it appears not to be), but I think preserving the existing node names is reasonable behavior given that they are being copied into a new graph.